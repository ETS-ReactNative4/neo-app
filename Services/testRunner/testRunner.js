import renderer from "react-test-renderer";

const testRunner = testCase => {
  test(testCase.title, () => {
    const tree = renderer.create(testCase.Component).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

export default testRunner;
