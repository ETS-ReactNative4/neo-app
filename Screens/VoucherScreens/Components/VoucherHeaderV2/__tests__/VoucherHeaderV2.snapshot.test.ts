import VoucherHeaderV2TestCases from "../VoucherHeaderTestCases";
import testRunner from "../../../../../Services/testRunner/testRunner";

VoucherHeaderV2TestCases.map(testRunner);
