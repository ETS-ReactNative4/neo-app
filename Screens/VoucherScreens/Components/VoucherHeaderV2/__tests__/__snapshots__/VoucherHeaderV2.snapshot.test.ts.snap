// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher Header 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Array [
      Object {
        "height": 214,
        "width": 750,
      },
      Object {},
    ]
  }
>
  <Image
    onError={[Function]}
    source={
      Object {
        "uri": "https://pyt-images.imgix.net/images/place-holder.png",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(0,0,0,0.6)",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.8}
      style={
        Object {
          "left": 0,
          "padding": 20,
          "position": "absolute",
          "top": 0,
        }
      }
    >
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 20,
            },
            undefined,
            Object {
              "fontFamily": "voyager",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </TouchableOpacity>
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 20,
          },
          undefined,
          Object {
            "fontFamily": "voyager",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "SourceSansPro-SemiBold",
          "fontSize": 18,
          "lineHeight": 25,
          "marginHorizontal": 24,
          "paddingTop": 12,
          "textAlign": "center",
        }
      }
    >
      Insurance Voucher
    </Text>
    <Image
      resizeMode="contain"
      source={
        Object {
          "testUri": "../../../assets/images/shapes/hiring_topic.png",
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 355,
          "position": "absolute",
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "white",
          "bottom": 0,
          "height": 16,
          "left": 0,
          "position": "absolute",
          "width": 356.7,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "white",
          "bottom": 0,
          "height": 16,
          "position": "absolute",
          "right": 0,
          "width": 355,
        }
      }
    />
  </View>
</View>
`;
